WISE-Community/WISE
# Add student scores + work (component grading) to milestone reports

Milestone reports (once the milestone's completion threshold has been met) now show two tabs below the report content: Recommendations (if authored) and Student Work. Student Work shows the node grading view for the component that the milestone addresses.

To test:
1. Run ```npm install``` to update dependencies.
1. Set up a run with a milestone report. (Here is a sample unit with a milestone report enabled: 
[212.zip](https://github.com/WISE-Community/WISE/files/4036817/212.zip)).
1. Add two or more students to the run and complete the milestone component.
1. Open the teacher tools for the run and verify that once the completion thresholds have been met, the report content and summary graphs appear for the milestone and that the Recommendations and Student Work tabs show up. Verify that the Student Work display shows c-rater score and work for each workgroup correctly.
1. From the teacher's milestone display, make sure sending annotations works. (Note that changing the score for a workgroup will remove that workgroup's data from the milestone report summary and graphs. This is because milestone reports only look at the latest student score and include it if the type is autoScore. I'm creating a separate issue to address this.)

Closes #2150.
